src/Tizen.NUI/src/public/Window.cs

```diff
@@ -274,6 +274,17 @@ internal Window(Rectangle windowPosition, string name, string className) : this(
             }
         }
 
+        /// <summary>
+        /// ViewConnected will be triggered when the view connected on Window
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler ViewConnected;
```

Comment:
sorry for late reply. please change this name as "ViewAdded" 
and, please add this attribute => 

        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
        [EditorBrowsable(EditorBrowsableState.Never)]